src/main/java/net/snowflake/ingest/SimpleIngestManager.java

```diff
@@ -564,13 +564,13 @@ public IngestResponse ingestFiles(
             requestId, pipe, files, showSkippedFiles, Optional.ofNullable(clientInfo));
 
     // send the request and get a response....
-    HttpResponse response = httpClient.execute(httpPostForIngestFile);
-
-    LOGGER.info(
-        "Attempting to unmarshall insert response - {}, with clientInfo - {}",
-        response,
-        clientInfo);
-    return ServiceResponseHandler.unmarshallIngestResponse(response, requestId);
+    try (CloseableHttpResponse response = httpClient.execute(httpPostForIngestFile)) {
```

Comment:
Could you update the four SS APIs as well?

Reply:
yes, let me do it..